From b859e9bbaf8beb121636bb15a567c7bef1e01248 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:42 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Update lasse_collin_pubkey.txt to the current copy at
https://tukaani.org/misc/lasse_collin_pubkey.txt, the URL the spec names
for it (same certificate(s)).

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 lasse_collin_pubkey.txt | 50 ++++++++++++++++++++---------------------
 1 file changed, 25 insertions(+), 25 deletions(-)

diff --git a/lasse_collin_pubkey.txt b/lasse_collin_pubkey.txt
index 4a391c6..733730c 100644
--- a/lasse_collin_pubkey.txt
+++ b/lasse_collin_pubkey.txt
@@ -13,18 +13,18 @@ CduUwSKv+6RmtWI5ZmTQYOcBRcZyGKml9X9Q8iSbm6cnpFXmLrNQwCJN+D3SiYGc
 MtbltZo0ysPMa6Xj5xFaYqWk/BI4iLb2Gs+ByGo/+a0Eq4XYBMOpitNniQARAQAB
 tCdMYXNzZSBDb2xsaW4gPGxhc3NlLmNvbGxpbkB0dWthYW5pLm9yZz6JAlEEEwEK
 ADsCGwMCHgECF4AECwkIBwMVCggFFgIDAQAWIQQ2kMJAzlG0Zw0wrRw47nV9aRhG
-IAUCZZwJyQUJGuHiNwAKCRA47nV9aRhGIE4qD/4jdFTe3WPpLgvz/jdlbnSZxr7q
-OS6H/ZJFENHO4SbavXdoXLtj+t6/lqWq890Js8IpWaaiJLowzW1xJMEg99W6k0KD
-3pHUbwPxf0GCSAt/W4JYxdTj+1ggdHjx5yBAmOakjnOH+ZDKQNBnDOI6ghf3ew+H
-9z/b0mQX3rlQbtoqSPZtuDOdFcjCOSwEyqdV+9eNqnv2CoKZkiGoUB1WGCbqKUkY
-KiUJ3WldmPQ5RQYjEi7zZWVac1VuwBA0XOku+W4cCJ5DnPyK7CtMwC84VvaodlOX
-UAK3Y5BIZpZM2Rk6yMX5lFDA5nA8UuHJQRDjTVmh3BIdgRvp0ZV6ogtqNE7RifpW
-aBWDIsCkimcbCJJM+edOLiVZog+ia1Ts8zu33wj7Tnvp5znLc8NLZIqwu1HKLS97
-m+Yf5oC3ObTZtXbVF+OglWe/3ljLHdL2bJxNdtcVlChSNPUW3fgLHk9Fzrlnqdab
-tSGwI/0Ryt00cKjRiMOagTn5Nly6boCtgGYdQafQoSrs3eQjnWVgbNYDMgPyl4k+
-Q5RJLEY7AvtXo7FUEgOTfr9PWmjmc2JzGpxbtwl6sQi6yLrBZTRf1Xao2OjOje6G
-XdUbXNmgOv16sWxcI0s4lX1z28BgHQfwXhBFBRjw2Sy+6TfFXjX24thcpMwvyJ3c
-xhMtdY4N4jyfRjYe8LkCDQRMxDmSARAAv8XAp2PGA/G1KmCrVIzOBm1NPIuqGAYP
+IAUCaWpSWgUJHqf7jwAKCRA47nV9aRhGIPwPD/9c+3D47ZR6u91BvwZAT1Lp0XvQ
+2pQzh/kf9x4y6uDWwf2oUEEHCEaj9TwtIrswSm0v2ut8jBdxghCfYLZDfs4UUN3d
+rjkDCSaH5tBGTrDn7HrlnCP+7FtcanpH9uRD1L8yYxJVO4M7lvvLAYmYOe2gn6Xd
+rjdFDAQmCLCD5oxNX6w4pbGWDE1mJomtLsUguXuxqsvutddWT/+EK1d9vb1ctxHd
+i4x/xJYJKWTfQ2rhKoUmRbt/LE9LrRtKgxeJ9FKwdzNkjkXdnt11mHs5dO+MQzSZ
+136PEm9Gls64+xZdzaZFbqB7oaNnFsXjAAe3G6Q2p4kn+F/4+i47O4L24DQ+2ug8
+CHcsMyR3217hSP8Fmf+Zklixpn1qAdEMzFXL8FofjEBNpA7yLFA7yZgL5MyvgwLD
+jr4CseXxvXBCav89jm5u5nr7XFTk8hyo08eCnBKKXK+6NpQEG7DmNvd7zqrcYNkb
+cquPxgNMCdEo8gFlogKaj5p4SQ9ux/Q71/2lqYM3b53vM+UZfGE4Ug/3uleLAMJx
+QzlOOdIvU9Uyz253GjaG64bAiaZ4eOrupGS6IyyfbN7ohtDnRkrb/T/RzWscUgqO
+eUzPyGsRx/Cnq5Nqf7sd8rIfArBMmP/vXxrrE/Ksj/H28ywMGP0qTb0XIHYr07zP
+8itwED4lN0kHgwtprLkCDQRMxDmSARAAv8XAp2PGA/G1KmCrVIzOBm1NPIuqGAYP
 c1l9p0dYdhEgvfw0NXcl5MDv1jbOPZ2PspA8NP7Rqp6LNNXYTeM/eIJDndU5Phyi
 ewFpACAp7Gmm2dL5PUOhu0gIUnQYbN/QdGPoo7bNI646K1Y9aVTBu9fszQssjb6G
 qXHSNM+pskVn9lropO1tLrF0I9VSlSphlCmiQRlzBCZSnxD6UagkPaw1gJnJqnrd
@@ -36,17 +36,17 @@ CjdACbWJsPEoIOrRFxY+NwJEA39Dkyalyh2l0qTNXTIYhLiDuzl+tWuBX+SjHavj
 9jGyvwr3T37gfzYCNMoZf8GaxAUJMCoGTqnsjTPGMion/DfdNkFDQ+fivdYiVQ9p
 /Njpr38sC83V8dHF/1KkIHImyzMPTdC7l/lMHyC2Gx2dWZOjuOOKit0Qoy3DZoQw
 vN1ZZND9M1UAEQEAAYkCPAQYAQoAJgIbDBYhBDaQwkDOUbRnDTCtHDjudX1pGEYg
-BQJlnAmyBQka4eIgAAoJEDjudX1pGEYguyYQAJo+5SnMMdu+d70mWfUb9PZg7P5C
-GRepHnckx9Sis5oR5s7NNl5j5Yy4J1UwsmrP+mn52ujqewkkVsCq65NGQQx7+tkw
-uKGvnGBkHdrI+aJk86qLMf4DlnNJEmN8t5jTGQfRLbFVf2I8EY6qXAzCSmL9Zs++
-rDUz65GOTB1EP0XmBRsuVYRfDbFezrPQH0JDucbXFi/2BDnl2/Mk9NBoQ0CvB4oG
-tLDiQZ+jV7n1VXXJ1faD9s7i0hOTdcG6rlyIqi/LyAzdCnOYTkmv3U1kdmzkvrh1
-KEiejnM5fj27RE2v191vh3hgZ+X5+uwjNTP0QC4qP8XykQOAA8usOMVZ72lyXCAk
-wiUcRdrAXLN/XbIFNcQ3m4d3W6t60Gk09wFlUKaEltDMlPUsxiSG3qFwFGPBP6UV
-h3mjJMAl1jltLrR7ybez0SczfrcAtdCsKTvgzV9W2TzUfK2R9PBanmXTXK2M7yU3
-IquHt3Je4aSP7XYb5D+ajlbFNvnXOYcai8WryfC5nLAfV4MbPX+UlRaYCqqHVhut
-gK93re1L5mMI3zjG5Ri5jLpUA9toSJCIJIY5zwr/8LL/ZL4TixXlouA17yjkpY/e
-Bjs8cNj1O3aM4jY2FKCS8UbfxOiARk/5kBMRPEZ/mqpMQttzE8KVjOv6fRxy/eVE
-888/gToe5kb8qYwy
-=6rZC
+BQJpalI+BQkep/uUAAoJEDjudX1pGEYgxkgP+wf9Kk6Qgt1+mqumeEd1iA2LrCM2
+w1UG12LRiM0fsTJZTxFBOt46J23YaD5wYoydaGpE/OS4ViXnZTI7vndpTr2KAI7Z
+aiNv+JgDA5x3olPVdMC9O0DdVCDXkLOxyJezsS1didGdujmsD9XX34Eou0ACgrJn
+QqYXIrN+r/SuxvRu7Yhx2M+zdXzch5PkYID2PXRUY+Nzz30Vpx1YJ5PUegjiB5qL
+NvldM7kOb12cOPVYlr3/sh5zfAcBNXcYFAr78buYZJ78X6PyQjIcsXPeZoKt4h/V
+R8UApYcmjfdkus6ClkrZqA/tL4CkanXGCwpmdj6P/Bqy3qu1n0GfIU7+wBnb3c7O
+8qpXg41PnNXZl9HOXCs25laf9XM0uuSl+TmdsJa54DxcUEMvN4C4RuKhIRXVRzKl
+jGCRZC++XgQ4/wM6xFufbu18UcBaZwm6luOf1vDqMosQtSSxt/tCLUFd1pfmilzi
+fospwFDVOfaOxtkN8tlJASAR1TmmL5ABmHDucbMtFpLteeeKUjXlOXviYqrYSTDX
+JL2T1dd1Nk/z5jAOaNHTs+/T8wQs84h7WT7aPYCAnR54ExBpsJ9160VgfKUyQQRi
+bIx8nCY8EgaT8c0L/L7LOWlNuCrdMM0TI+fRx/2LIwVBEYXQt1rMzeMmeJTSnGRG
+x39xjsY5atKw0oo8
+=iJCM
 -----END PGP PUBLIC KEY BLOCK-----
-- 
2.56.0


